Apps/W1/HybridGP/app/src/pages/HybridGPErrorsOverviewFb.page.al

```diff
@@ -4,18 +4,29 @@ using Microsoft.DataMigration;
 
 page 40132 "Hybrid GP Errors Overview Fb"
 {
-    Caption = 'GP Upgrade Errors';
+    Caption = 'GP Migration Overview';
     PageType = CardPart;
-    InsertAllowed = false;
-    DelayedInsert = false;
-    ModifyAllowed = false;
-    SourceTable = "GP Migration Error Overview";
```

Comment:
We cannot drop a source table, it is a breaking change. To handle it we need to replace the FB and introduce a new one, or keep the factbox. The better solution is probably to introduce a new page and to make the existing page as not visible. 

Looking at this PR, there is no need to remove the source table, you can add the new information regardless.